app/locale/en_US/template/email/contact_form_auto_reply.html

```diff
@@ -0,0 +1,13 @@
+<!--@subject Contact Form Confirmation @-->
+<!--@vars
+{"var data.name":"Sender Name",
+"var data.email":"Sender Email",
+"var data.telephone":"Sender Telephone",
+"var data.comment":"Comment"}
+@-->
+
+Thank you for contacting us, we will reply you shortly.<br>
+Have a nice day.
+<br><br>
+In case you received this message without using the contact form on our website, please ignore it.<br>
+Thank you for your understanding.
```

Comment:
Thank you for your interest in the products sold by our company. This confirmation message is sent automatically as a result of your request made through the contact form. We will reply you shortly.

If someone else made the request on your behalf abusively, please ignore and delete our messages. Thank you for understanding.

Reply:
In my opinion this is way too much, at the moment I prefer the one that's already in the PR.